docs/how-to/using-git.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using Git
+---
+
+Using Git
+=============
+
+Git is used as version control system for the source code in the project.
+
+
+
+## Basic principles
+Whenever the devleoper makes any code changes, a new Pull Request (PR) should be created. The new PR should branch out from the main branch.
```

Comment:
developer

Reply:
Also, not sure if I agree that a new PR should always branch out from main. It can be very useful to branch out separate issues from a topic branch and create a PR into the given branch instead of just making a single (huge) PR that contains multiple features/fixes.